canonical-websites/snapcraft.io
# Add countries to listing template

# Summary

Fixes #1073 
Pass list of countries for listing template, this will allow us to build the blacklist and whitelist form

# QA

- in file `templates/publisher/listing.html` add: 
```
    {% for country in countries %}
       {{ country }}
    {% endfor %}
```
- `./run`
- http://127.0.0.1:8004/account/snaps/toto/listing
- Make sure you have the list of countries

# Note
When building the form we will need to keep the `alpha_2` and the `name` of every country. The frontend will have to push a list of `alpha_2` for the whitelist of the blacklist. The post endoint is already handled.
